### PR TITLE
WF-62505-Update chart FAQ-Custom datapoints for series

### PR DESCRIPTION
**Pull request by YuvarajPalanisamy**
Added more information and sample

### CODE REVIEW

File: WindowsForms/chart/FAQ/How-to-customize-the-data-points-for-Chart-Series.md [hunk 1]
```diff
@@ -9,26 +9,155 @@ documentation: ug
...
-series1.SeriesModelImpl = New NonIndexedModel(New Double() {2, 5, 3, 4, 6}, New Double() {10, 33,20, 43, 12})
+series1.SeriesModel = New NonIndexedModel(New Double() {2, 5, 3, 4, 6}, New Double() {10, 33, 20, 43, 12})
 
 {% endhighlight %}
 
-{% endtabs %}	
\ No newline at end of file
+{% endtabs %}	
+
+You can find the complete sample from this [link](https://www.syncfusion.com/downloads/support/directtrac/general/ze/Chart-CustomDataPoints215363699).
```
Comment (Manivannan-E) [inline WindowsForms/chart/FAQ/How-to-customize-the-data-points-for-Chart-Series.md:157]: @YuvarajPalanisamy remove the sample link and its content.

Comment (YuvarajPalanisamy) [reply WindowsForms/chart/FAQ/How-to-customize-the-data-points-for-Chart-Series.md]: @Manivannan-E , I have removed the Sample link and it's content. Please review and merge.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Review by SyncfusionKarthikeyan — APPROVED** [review]
This is fine

**Pull request merged**